WeightProducer/python/SVJxsecs.py

```diff
@@ -145,5 +145,56 @@
     "mZprime-5100" : {
         "CrossSection" : XSValues(XS_13TeV=0.0001523),
     },
+    # Full t-channel cross section
+    "t-channel_mMed-200" : {
+        "CrossSection" : XSValues(XS_13TeV=990.8), # uncertainty: 3.617
+    },
+    # "t-channel_mMed-400" : {
```

Comment:
Why are these commented out?

Reply:
Okay, I've added this info in a comment. For the UL signal production, we should come up with a better naming scheme to avoid this ambiguity.